crates/nu-command/src/strings/ansi/ansi_.rs

```diff
@@ -425,17 +425,38 @@ static CODE_LIST: LazyLock<Vec<AnsiCode>> = LazyLock::new(|| { vec![
     AnsiCode { short_name: Some("grey89"), long_name: "xterm_grey89", code: Color::Fixed(254).prefix().to_string()},
     AnsiCode { short_name: Some("grey93"), long_name: "xterm_grey93", code: Color::Fixed(255).prefix().to_string()},
 
-    // Attributes
+    // Attribute, SGR (Select Graphic Rendition) style codes
     AnsiCode { short_name: Some("n"), long_name: "attr_normal", code: Color::Green.suffix().to_string()},
     AnsiCode { short_name: Some("bo"), long_name: "attr_bold", code: Style::new().bold().prefix().to_string()},
     AnsiCode { short_name: Some("d"), long_name: "attr_dimmed", code: Style::new().dimmed().prefix().to_string()},
     AnsiCode { short_name: Some("i"), long_name: "attr_italic", code: Style::new().italic().prefix().to_string()},
     AnsiCode { short_name: Some("u"), long_name: "attr_underline", code: Style::new().underline().prefix().to_string()},
+
+    // NOTE: Most modern terminals ignore SGR codes 5 & 6 as they are considered distracting or even 
+    // seizure-triggering for some users. We're including them here for completeness and compatibility with older terminals.
     AnsiCode { short_name: Some("bl"), long_name: "attr_blink", code: Style::new().blink().prefix().to_string()},
+    AnsiCode { short_name: Some("bf"), long_name: "attr_blink_fast", code: "\x1b[6m".to_owned()},
+
+    AnsiCode { short_name: Some("re"), long_name: "attr_reverse", code: "\x1b[7m".to_owned()},
```

Comment:
This one should probably use the nu-ansi-term syntax since it's supported
```suggestion
    AnsiCode { short_name: Some("re"), long_name: "attr_reverse", code: Style::new().reverse().prefix().to_string()},
```